clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/CTInAppNativeInterstitialFragment.java

```diff
@@ -61,6 +63,16 @@
     private FrameLayout videoFrameInDialog;
 
     private ViewGroup.LayoutParams imageViewLayoutParams;
+    private final OnBackPressedCallback onBackPressedCallback = new OnBackPressedCallback(false) {
+        @Override
+        public void handleOnBackPressed() {
+            if (exoPlayerFullscreen) {
+                config.getLogger().debug("Frag", "handleOnBackPressed() called");
```

Comment:
A more descriptive tag to this log could be more useful

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/dc0283ab69ceb9e8f9593826d016f8decb076280